ReactWindows/ReactNative.Shared/DevSupport/DisabledDevSupportManager.cs

```diff
@@ -65,10 +70,17 @@ public string JavaScriptBundleUrlForRemoteDebugging
 
         public async void HandleException(Exception exception)
         {
+#if WINDOWS_UWP
             await CoreApplication.MainView.CoreWindow.Dispatcher.RunAsync(CoreDispatcherPriority.High, () =>
             {
                 ExceptionDispatchInfo.Capture(exception).Throw();
             }).AsTask().ConfigureAwait(false);
+#else
+            await Dispatcher.CurrentDispatcher.InvokeAsync(() =>
+            {
+                ExceptionDispatchInfo.Capture(exception).Throw();
+            }, DispatcherPriority.Send).Task.ConfigureAwait(false);
```

Comment:
`DispatcherPriority.Send` appears to be the closest equivalent to `CoreDispatcherPriority.High`